loom/graphics/gfxStateManager.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * ===========================================================================
+ * Loom SDK
+ * Copyright 2011, 2012, 2013
+ * The Game Engine Company, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ===========================================================================
+ */
+
+#pragma once
+
+
+
+#ifdef __cplusplus
+extern "C" {
+#else
+#include <stdbool.h>
+#endif
+
+enum Graphics_GLState {
+    GFX_OPENGL_STATE_NANOVG = 0,
+    GFX_OPENGL_STATE_QUAD   = 1,
+    GFX_OPENGL_STATE_MAX    = 2,
+};
+
+bool Graphics_IsGLStateValid(enum Graphics_GLState state);
```

Comment:
Can you please add some docs to these functions? Also, to match surrounding conventions, does it make sense to namespace this stuff? Or should it remain C-compatible for easier integration?